src/module/dialog-flow.component.ts

```diff
@@ -14,7 +14,7 @@ export class DialogFlowService {
 
 		const matchedHandlers = [this.handlers.get(intent), this.handlers.get(action)].filter(v => v);
 
-		if (!matchedHandlers) {
+		if (matchedHandlers.length === 0) {
```

Comment:
`matchedHandlers` can't be null either, it's at least an empty array, you can easily use `if (!matchedHandlers)`